Anytype/Sources/ServiceLayer/Block/BlockActionHandler/BlockActionHandler.swift

```diff
@@ -130,13 +130,12 @@ final class BlockActionHandler: BlockActionHandlerProtocol {
         service.delete(blockIds: blockIds)
     }
     
-    func moveToPage(blockId: String, pageId: String) {
+    func moveToPage(blockId: String, pageId: String) async throws {
         AnytypeAnalytics.instance().logMoveBlock()
-        Task {
-            try await blockService.moveToPage(objectId: document.objectId, blockId: blockId, pageId: pageId)
-        }
+        try await blockService.moveToPage(objectId: document.objectId, blockId: blockId, pageId: pageId)
```

Comment:
moveToPage support multiple block move. Is better for use one call for prevent multiple animaitons in editor

Reply:
Sure sounds good. Just so I understand, are you suggesting that we make the block service capable of moving multiple blocks in a single method call?

Reply:
I see what you meant, the client command already accepts an array of block ids. Block service now accepts an array of block ids (see dca6160e6cc74e6e78aa90fcbcd7e8d902370789), thanks for the tip!